src/webrtc/call.ts

```diff
@@ -2106,7 +2110,7 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
                 this.callLengthInterval = setInterval(() => {
                     this.callLength++;
                     this.emit(CallEvent.LengthChanged, this.callLength);
-                }, 1000);
+                }, CALL_LENGTH_INTERVAL);
```

Comment:
Hmm, this feels a bit weird and is essentially defining the number of milliseconds in a second. Also, a setInterval won't execute after exactly the amount of time specified (especially when browsers do funny things with suspending timer execution in background pages), so this might have a tendency to drift less accurate over time.

I'd probably just store the timestamp when the call started and recalculate the length based on `now - startTime` each time, then this constant really is just the update interval.

Reply:
Good point!